openedx_learning/apps/authoring/publishing/api.py

```diff
@@ -771,6 +774,64 @@ def create_container_version(
     return container_version
 
 
+class ChildrenEntitiesAction(Enum):
+    """Possible actions for children entities"""
+
+    APPEND = "append"
+    REMOVE = "remove"
+    REPLACE = "replace"
+
+
+def get_next_entity_list(
```

Comment:
`get_` implies a read operation, when this is actually creating something.

Reply:
Updated: e1222ea27274